projects/addon-editor/components/editor-socket/editor-socket.component.ts

```diff
@@ -11,12 +10,11 @@ import {
     SecurityContext,
     ViewEncapsulation,
 } from '@angular/core';
-import {TUI_EDITOR_STYLES} from '@taiga-ui/addon-editor/tokens';
 import {TUI_SANITIZER} from '@taiga-ui/core';
 
 // @dynamic
 @Component({
-    selector: 'tui-editor-socket',
+    selector: 'tui-editor-socket,[tuiEditorSocket]',
```

Comment:
Do we really need two selectors? TipTap cannot work with custom tag as host?

Reply:
removed